cmd/nfd-master/main.go

```diff
@@ -111,7 +111,9 @@ func main() {
 
 	// Get new NfdMaster instance
 	args.GrpcHealthPort = GrpcHealthPort
-	instance, err := master.NewNfdMaster(master.WithArgs(args))
+	instance, err := master.NewNfdMaster(
+		master.WithArgs(args),
+	)
```

Comment:
nit: kinda noise, irrelevant. Suggest to keep it as it was

```suggestion
	instance, err := master.NewNfdMaster(master.WithArgs(args))
```